git/tests/test.bats

```diff
@@ -3,3 +3,10 @@
   actual_version="$(hab pkg exec "${TEST_PKG_IDENT}" git --version | awk '{print $3}')"
   diff <(echo "$actual_version") <( echo "${TEST_PKG_VERSION}")
 }
+
+@test "Init a git repo" {
+  run hab pkg exec "${TEST_PKG_IDENT}" git init git_test_repo
+  [ $status -eq 0 ]
+  [ -d git_test_repo/.git ]
+  rm -rf git_test_repo
```

Comment:
It should be fine, but if we wanted we could so something like:

```suggestion
  TEST_TMP_REPO_NAME="git_test_repo-${RANDOM}-${RANDOM}"
  run hab pkg exec "${TEST_PKG_IDENT}" git init "${TEST_TMP_REPO_NAME}"
  [ $status -eq 0 ]
  [ -d "${TEST_TMP_REPO_NAME}/.git" ]
  rm -rf "${TEST_TMP_REPO_NAME}"
```

Reply:
This shouldn't be necessary. We're running tests in a clean studio each time.